File: 02 Datentypen/Pascal/Aufgabe8.pas
Program Aufgabe8;

{$codepage utf8}

Const MwSt: real = 0.19;

Var Preis, ArtikelMwSt, Netto: real;

Begin
    Write('Bitte gib den Preis deines Artikels ein: ');
    ReadLn(Preis);
    WriteLn(); // Leere Zeile ausgeben, Write('\n'); geht auch.

    ArtikelMwSt := Preis * MwSt;
    Netto := Preis - ArtikelMwSt;

    WriteLn('Der Artikel hat ', ArtikelMwSt:1:2, ' Euro Mehrwertsteuer und sein Nettopreis ist ', Netto:1:2, ' Euro.');
End.

File: 02 Datentypen/Go/Aufgabe8.go
package main

import "fmt"

const mwSt float32 = 0.19

var (
	preis       float32
	artikelMwSt float32
	netto       float32
)

func main() {
	fmt.Print("Bitte gib den Preis deines Artikels ein: ")
	fmt.Scan(&preis)

	fmt.Println() // Leere Zeile ausgeben, fmt.Print('\n'); geht auch.

	artikelMwSt = preis * mwSt
	netto = preis - artikelMwSt

	fmt.Println("Der Artikel hat", artikelMwSt, "Euro Mehrwertsteuer und sein Nettopreis ist", netto, "Euro.")
}
